client/landing/stepper/hooks/use-user-can-manage-options.ts

```diff
@@ -1,27 +1,25 @@
-import { useSelect } from '@wordpress/data';
-import { useSelector } from 'calypso/state';
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'calypso/state';
 import { canCurrentUser } from 'calypso/state/selectors/can-current-user';
-import isRequestingSites from 'calypso/state/sites/selectors/is-requesting-sites';
-import { SITE_STORE } from '../stores';
-import { useSiteSlugParam } from './use-site-slug-param';
-import type { SiteSelect } from '@automattic/data-stores';
+import { requestSite } from 'calypso/state/sites/actions';
+import { useSiteData } from './use-site-data';
 
 export function useCanUserManageOptions() {
-	const siteSlug = useSiteSlugParam();
-	const siteId = useSelect(
-		( select ) => siteSlug && ( select( SITE_STORE ) as SiteSelect ).getSiteIdBySlug( siteSlug ),
-		[ siteSlug ]
-	);
-	const isRequesting = useSelector( ( state ) => isRequestingSites( state ) );
```

Comment:
This hook relies whether we're requesting sites but it doesn't request on its own 🥲